<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1044</width>
    <height>704</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="minimumSize">
    <size>
     <width>1044</width>
     <height>677</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>1044</width>
     <height>677</height>
    </size>
   </property>
   <property name="cursor">
    <cursorShape>CrossCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">QWidget{
setbackground color : gray}</string>
   </property>
   <widget class="QLineEdit" name="TextInput">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>210</y>
      <width>1021</width>
      <height>41</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>71</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Words :</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_WordsNum">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>0</y>
      <width>121</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="wordWrap">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>50</y>
      <width>81</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Accuracy:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_Accuracy">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>45</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_chooseText">
    <property name="geometry">
     <rect>
      <x>940</x>
      <y>0</y>
      <width>101</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Choose Text</string>
    </property>
   </widget>
   <widget class="QComboBox" name="ChangeLang">
    <property name="geometry">
     <rect>
      <x>570</x>
      <y>0</y>
      <width>131</width>
      <height>32</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>English</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Belarusian</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Russian</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Arabic</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>German</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Hebrew</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Chineese</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>French</string>
     </property>
    </item>
   </widget>
   <widget class="QPushButton" name="startButton">
    <property name="geometry">
     <rect>
      <x>720</x>
      <y>0</y>
      <width>100</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Start</string>
    </property>
   </widget>
   <widget class="QPushButton" name="stopButton">
    <property name="geometry">
     <rect>
      <x>720</x>
      <y>40</y>
      <width>100</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Stop</string>
    </property>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>9</x>
      <y>269</y>
      <width>1031</width>
      <height>401</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="keyboard"/>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>70</y>
      <width>91</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Text for typing</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>10</y>
      <width>58</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Time:</string>
    </property>
   </widget>
   <widget class="QLabel" name="timer">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>10</y>
      <width>101</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QTextEdit" name="TextToWrite">
    <property name="geometry">
     <rect>
      <x>13</x>
      <y>100</y>
      <width>1011</width>
      <height>101</height>
     </rect>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>40</y>
      <width>131</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Symbols per second:</string>
    </property>
   </widget>
   <widget class="QLabel" name="symPerSecond">
    <property name="geometry">
     <rect>
      <x>337</x>
      <y>39</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
